web/src/people.ts

```diff
@@ -1501,6 +1508,12 @@ export function make_user(user_id: number, email: string, full_name: string): Us
         // This property allows us to distinguish actual server person
         // objects from fake person objects generated by this function.
         is_missing_server_data: true,
+
+        // Used to differentiate from PseudoMentionUser in `typeahead_helper`.
+        is_broadcast: false,
```

Comment:
Not all user objects come from `make_user`, so we can’t just make up fictional fields here.